src/averageAges.js

```diff
@@ -15,11 +15,14 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const filterCallback = people.filter(man => century
```

Comment:
```suggestion
  const men = people.filter(man => century
```
use clear naming for variables - this array includes men (filtered by century or not)